.snyk

```diff
@@ -0,0 +1,50 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.25.0
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+   SNYK-JAVA-COMMONSIO-1277109:
+    - '*':
+        reason: We need to update to Vertx 4.x
+        expires: 2025-12-31T05:00:40.733Z
```

Comment:
@ksclarke: these expiration dates seems a bit far in the future to me, should we make them sooner?